lectures/ifp_egm.md

```diff
@@ -102,32 +101,35 @@ The following function stores default parameter values for the income
 fluctuation problem and creates suitable arrays.
 
 ```{code-cell} ipython3
-def ifp(R=1.01,             # gross interest rate
-        β=0.99,             # discount factor
-        γ=1.5,              # CRRA preference parameter
-        s_max=16,           # savings grid max
-        s_size=200,         # savings grid size
-        ρ=0.99,             # income persistence
-        ν=0.02,             # income volatility
-        y_size=25):         # income grid size
-  
+Household = namedtuple('Household', ('β', 'R', 'γ', 's_size', 'y_size', \
+                                     's_grid', 'y_grid', 'P'))
+
+def create_household(R=1.01,             # gross interest rate
+                     β=0.99,             # discount factor
+                     γ=1.5,              # CRRA preference parameter
+                     s_max=16,           # savings grid max
+                     s_size=200,         # savings grid size
+                     ρ=0.99,             # income persistence
+                     ν=0.02,             # income volatility
+                     y_size=25):         # income grid size
+    
     # Create income Markov chain
     mc = qe.tauchen(y_size, ρ, ν)
     y_grid, P = jnp.exp(mc.state_values), mc.P
+    
     # Shift to JAX arrays
     P, y_grid = jax.device_put((P, y_grid))
     
     s_grid = jnp.linspace(0, s_max, s_size)
-    sizes = s_size, y_size
     s_grid, y_grid, P = jax.device_put((s_grid, y_grid, P))
 
     # require R β < 1 for convergence
     assert R * β < 1, "Stability condition violated."
-        
-    return (β, R, γ), sizes, (s_grid, y_grid, P)
+    
+    return Household(β=β, R=R, γ=γ, s_size=s_size, y_size=y_size, \
+               s_grid=s_grid, y_grid=y_grid, P=P)
```

Comment:
```suggestion
    return Household(β=β, R=R, γ=γ, s_size=s_size, y_size=y_size, 
               s_grid=s_grid, y_grid=y_grid, P=P)
```

Reply:
Nice catch @HumphreyYang .  Not needed after comma.